test/tests/meshgenerators/break_mesh_by_block_generator/hanging_nodes_parallel.i

```diff
@@ -0,0 +1,69 @@
+[GlobalParams]
+  displacements = "disp_x disp_y disp_z"
+[]
+
```

Comment:
```suggestion
```

Reply:
if I remove this block it doesn't hit the assertion

Reply:
moved displacements inside the mesh_block